### ISSUE TITLE
Error CS0029 Cannot implicitly convert type 'CommandLine.ParserResult<Options>' to 'Options'

### ISSUE THREAD

**Issue by Bluscream** [commandlineparser/commandline#511]
```c#
using CommandLine;
using System;
using System.Windows.Forms;

namespace MWR_Config_Editor
{
    static class Program
    {
        private static readonly NLog.Logger Logger = NLog.LogManager.GetCurrentClassLogger();
        public static Options Arguments;
        public class Options
        {
            [Option('c', "console", Required = false, HelpText = "Enable console")]
            public bool Verbose { get; set; }
        }
        /// <summary>
        /// The main entry point for the application.
        /// </summary>
        [STAThread]
        static void Main(string[] args)
        {
            Logger.Debug("Program started");
            Arguments = Parser.Default.ParseArguments<Options>(args);
            Application.EnableVisualStyles();
            Application.SetCompatibleTextRenderingDefault(false);
            Application.Run(new Main());
            Logger.Debug("Program ended");
        }
    }
}
```

**Comment by moh-hassan** [comment]
modify:

          public static Options Arguments;
to

         public static ParserResult<Options> Arguments;

**Comment by Bluscream** [comment]
And how am i supposed to access the variables inside it then? ParserResult does not contain a instance of Options

**Comment by IB38** [comment]
You need to use WithParsed, something like:
`Parser.Default.ParseArguments<Options>(args).WithParsed(result => Arguments = result);`

**Comment by moh-hassan** [comment]
The [Pr #634) exposes The Parsed option Value to the base class, available in develop branch and will be  released in  v2.9. 
You can:
```cs
var result = Parser.Default.ParseArguments<Options>(args);
Arguments = result.Value;
```

**Comment by Bluscream** [comment]
Thanks :)
